README.md

```diff
@@ -22,6 +22,26 @@ The following major GBA features are still missing from the crate:
 * Interrupt Handling
 * Serial Communication
 
+## Build Dependencies
+
+Install required cargo packages
+```sh
+rustup install nightly
+rustup +nightly component add rust-src
+cargo install cargo-make
+cargo install gbafix
+```
+
+Install arm build tools
+* Ubuntu
+  ```shell
+  sudo apt-get install binutils-arm-none-eabi
+  ```
+* OSX
+  ```shell
+  brew install --cask gcc-arm-embedded
+  ```
```

Comment:
This should also say something for windows. We can link to the download page on the ARM website: https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads

Reply:
Updated!